lib/jekyll/excerpt.rb

```diff
@@ -169,12 +177,11 @@ def liquid_block?(tag_name)
 
     def print_build_warning
       Jekyll.logger.warn "Warning:", "Excerpt modified in #{doc.relative_path}!"
-      Jekyll.logger.warn "",
-                         "Found a Liquid block containing separator '#{doc.excerpt_separator}'" \
-                         " and has been modified with the appropriate closing tag."
-      Jekyll.logger.warn "",
-                         "Feel free to define a custom excerpt or excerpt_separator in the" \
-                         " document's Front Matter if the generated excerpt is unsatisfactory."
+      Jekyll.logger.warn "", "Found a Liquid block containing the excerpt separator" \
+                         " #{doc.excerpt_separator.inspect}. "
+      Jekyll.logger.warn "", "The block has been modified with the appropriate closing tag."
+      Jekyll.logger.warn "", "Feel free to define a custom excerpt or excerpt_separator in the"
+      Jekyll.logger.warn "", "document's Front Matter if the generated excerpt is unsatisfactory."
```

Comment:
These are all the same warning, not separate messages, correct?

Reply:
@pathawks Yes. These *calls* are for the same warning.
I split them out into separate calls so that the messages appear to be formatted on the CLI.

I can't use a heredoc to emulate the formatting because Jekyll's `logger` squashes all whitespace into a single `space` char, which causes heredoc messages to simply print as a single line message.